includes/Core/Admin/Pointer.php

```diff
@@ -90,6 +93,28 @@ public function get_title() {
 		return $this->args['title'];
 	}
 
+	/**
+	 * Gets the pointer buttons.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return string|null Pointer buttons or null.
+	 */
+	public function get_buttons() {
+		return isset( $this->args['buttons'] ) ? $this->args['buttons'] : null;
```

Comment:
I think the `isset` condition isn't necessary here – at least the key should always exist. See other getters above.